bin/bootstrap.sh

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env bash
+
+# Define function sed_inplace that works on both GNU and BSD sed
+function sed_inplace() {
+    local file=$2
+    local command=$1
+    local tmp_file=$(mktemp)
+
+    sed -e "${command}" "${file}" >"${tmp_file}"
+    mv "${tmp_file}" "${file}"
+}
+
+# Retrieve the directory path from the first argument
+directory=$1
+
+# Check if the directory exists
+if [ ! -d "$directory" ]; then
+    echo "The directory '$directory' does not exist."
+    exit 1
+fi
+
+var_files=("$directory/variables.tf" "$directory/variables-common.tf")
+tfvars_file="$directory/terraform.tfvars"
+
+# Ensure docker is installed and running
+read -p "Please make sure you have docker installed and running. Press any key to continue..." -n 1 -r -s
+echo
+
+# Create and clean the $all_variables file
+all_variables=$(mktemp)
+
+# Concatenate the contents of the elements of var_files into a single file
+for var_file in "${var_files[@]}"; do
+    if [ -f "$var_file" ]; then
+        cat "$var_file" >>$all_variables
```

Comment:
```suggestion
        cat "$var_file" >>$all_variables
        echo "" >>$all_variables
```
Il faut rajouter une nouvelle ligne à chaque fichier sinon on se retrouve avec un pattern du style : 
```
variable "k8s_nodepool_autoscale" {
  type        = bool
  description = "Enable autoscaling feature (WIP)"
  default     = false
}variable "argocd_hostname" {
  type        = string
  description = "The hostname to use for the ArgoCD ingress"
}
```